lib/datasource/sbt/index.ts

```diff
@@ -9,7 +9,7 @@ async function resolvePackageReleases(
   artifact: string,
   scalaVersion: string
 ): Promise<string[]> {
-  const indexContent = await downloadHttpProtocol(searchRoot, 'sbt');
+  const indexContent = await downloadHttpProtocol(searchRoot + '/', 'sbt');
```

Comment:
Is there any concern here of what happens if the `searchRoot` already had a trailing slash?